pandas/plotting/_core.py

```diff
@@ -2172,7 +2172,7 @@ def boxplot(data, column=None, by=None, ax=None, fontsize=None,
         column = 'x'
 
     def _get_colors():
-        return _get_standard_colors(color=kwds.get('color'), num_colors=1)
+        return _get_standard_colors(color=kwds.get('color'), num_colors=3)
```

Comment:
why is this hardcoded?

Reply:
Right below this, the function `maybe_color_bp` references the first and third values of the array. It was working before with `num_colors=1` because of the bug, which returned 10 colors.